contracts/cw1-subkeys/.cargo/config

```diff
@@ -3,4 +3,4 @@ wasm = "build --release --target wasm32-unknown-unknown"
 wasm-debug = "build --target wasm32-unknown-unknown"
 unit-test = "test --lib"
 integration-test = "test --test integration"
-schema = "run --example schema"
+schema = "run --bin schema"
```

Comment:
In the end, the only real effect is changing `--example` to `--bin`?
Or does this have other positive improvements? Or needed as a future building block?

Reply:
Well, it also removes the schema gen code from the 'examples' directory so that a person new to CosmWasm contracts doesn't have to wonder "what the hell?" when looking at the source of one and trying to figure out what's what.

Reply:
Recently Dan mentioned this seemed odd and asked if we can rename the `examples` directory. When I first looked at how schemas are generated in contracts, I also had a "what the hell" moment.

Reply:
Okay, no blocker my side. Just seemed odd. But I guess I made this weird setup and an used to it.

If it makes CosmWasm users happy, let's do it.